builder/xenserver/common/common_config.go

```diff
@@ -18,11 +18,11 @@ type CommonConfig struct {
 	Password string `mapstructure:"remote_password"`
 	HostIp   string `mapstructure:"remote_host"`
 
-	VMName        string   `mapstructure:"vm_name"`
-	VMDescription string   `mapstructure:"vm_description"`
-	SrName        string   `mapstructure:"sr_name"`
-	FloppyFiles   []string `mapstructure:"floppy_files"`
-	NetworkName   string   `mapstructure:"network_name"`
+	VMName        string            `mapstructure:"vm_name"`
+	VMDescription string            `mapstructure:"vm_description"`
+	SrName        string            `mapstructure:"sr_name"`
+	FloppyFiles   []string          `mapstructure:"floppy_files"`
+	NetworkName   map[string]string `mapstructure:"network_name"`
```

Comment:
ok. Im outside and will make these changes after get home.

???? iPhone

? 2016?9?2??18:34?Rob Dobson <notifications@github.com<mailto:notifications@github.com>> ???


In builder/xenserver/common/common_config.go<https://github.com/xenserver/packer-builder-xenserver/pull/51#discussion_r77326388>:

> @@ -18,11 +18,11 @@ type CommonConfig struct {
>       Password string `mapstructure:"remote_password"`
>       HostIp   string `mapstructure:"remote_host"`
>
> -     VMName        string   `mapstructure:"vm_name"`
> -     VMDescription string   `mapstructure:"vm_description"`
> -     SrName        string   `mapstructure:"sr_name"`
> -     FloppyFiles   []string `mapstructure:"floppy_files"`
> -     NetworkName   string   `mapstructure:"network_name"`
> +     VMName        string            `mapstructure:"vm_name"`
> +     VMDescription string            `mapstructure:"vm_description"`
> +     SrName        string            `mapstructure:"sr_name"`
> +     FloppyFiles   []string          `mapstructure:"floppy_files"`
> +     NetworkName   map[string]string `mapstructure:"network_name"`


Can we please call this variable NetworkNames (given it's a map of networks).

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/xenserver/packer-builder-xenserver/pull/51/files/d956c2dd996c3afa1554cb8d0cc9ad00554d8adf#r77326388>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AKylpSrtij2JzdA4gB3rcD98WKSRD4hiks5ql_uDgaJpZM4JydPy>.